src/validation/managing.officer.validation.ts

```diff
@@ -66,6 +67,15 @@ export const updateManagingOfficerIndividual = [
   ...managingOfficerIndividual,
   ...start_date_validations,
   body("is_still_mo").not().isEmpty().withMessage(ErrorMessages.SELECT_IF_STILL_MANAGING_OFFICER),
+  body("start_date-day")
```

Comment:
I think this comes under "start_date_validations" shown above if i'm not mistaken, might not need to have this